addon/components/array-container.js

```diff
@@ -2,61 +2,57 @@ import 'ember-frost-bunsen/typedefs'
 
 import _ from 'lodash'
 import Ember from 'ember'
+const {A, Component} = Ember
 import computed, {readOnly} from 'ember-computed-decorators'
-
-import layout from './template'
 import PropTypeMixin, {PropTypes} from 'ember-prop-types'
-import {getLabel} from '../utils'
+import {deemberify, getLabel} from '../utils'
```

Comment:
I was hoping that with the `redux` implementation, that we could get away from `deemberify`. What are your thoughts on that?